doc/adr/0001-use-micro-frontends.md

```diff
@@ -0,0 +1,31 @@
+# 1. Use micro frontends
+
+Date: 2020-03-25
+
+## Status
+
+Proposed
+
+## Context
+
+We have designed the API layer (known as the backend) to be a custom set of services. This allows the user to tailor it
+to their own requirements, and introduce custom code as needed.
+
+We are currently planning to make the presentation layer (known as the frontend) a monolith. That is, a single service
+that generates all the HTML pages.
+
+As a monolith, it is hard, if not impossible, to change or replace parts without having to replace the whole. This will
+mean a lot of configuration options to support differing requirements. It will also limit adoption of the platform as it
+can't meet them.
+
```

Comment:
We've had feedback in the past that these ADRs seem to come from nowhere, even though we've clearly done some discovery. Can we link to the discovery ticket in here as that adds a good amount of context for the interested reader.